tests/extension/extension.test.ts

```diff
@@ -11,6 +11,12 @@ vi.mock('../../src/extension/grpc/client', () => ({
 }))
 
 test('initializes all providers', async () => {
+  const configValues = {
+    binaryPath: '.bin/runme'
+  }
+  vi.mocked(workspace.getConfiguration).mockReturnValue({
+    get: vi.fn().mockImplementation((config: string) => configValues[config])
```

Comment:
```suggestion
    get: vi.fn((config: string) => configValues[config])
```

FYI you can pass the implementation as an argument to `vi.fn`!

Reply:
Fixed